app/controllers/external_link_controller.rb

```diff
@@ -0,0 +1,10 @@
+class ExternalLinkController < ApplicationController
+  layout nil
+
+  def show
+    person = Person.find(params[:person_id])
+    link_name = params[:link_name]
+    url = person.send("#{link_name}_url")
+    redirect_to url, allow_other_host: true, status: :found
```

Comment:
What if we manually enter the name of a field in the URL bar that does not exist?